test/xapi/track_test.rb

```diff
@@ -2,6 +2,16 @@
 require_relative '../../lib/xapi'
 
 class TrackTest < Minitest::Test
+  def assert_archive_contains(filenames, zip)
+    files = []
+    Zip::InputStream.open(zip) do |io|
+      while (entry = io.get_next_entry)
+        files << entry.name
+      end
+    end
+    assert_equal filenames.sort, files.sort
+  end
```

Comment:
I really like that you've extracted `assert_archive_contents` into a method. :+1: